src/winfetch.ps1

```diff
@@ -24,470 +22,420 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-# Summary: Winfetch is a command-line system information utility for Windows written in PowerShell. 
```

Comment:
May I ask why the help section is removed?

Reply:
I changed it to use the standard comment-based help system so you could do `Get-Help filename.ps1`  You could also apply a URL to this and add `-Online` to get directed to the github readme.

Reply:
That's good. But lets still keep the help flag. :)